externs/olx.js

```diff
@@ -5053,6 +5118,22 @@ olx.source.VectorOptions.prototype.url;
 
 
 /**
+ * By default, an RTree is used as spatial index. When features are removed and
+ * added frequently, and the total number of features is low, setting this to
+ * `false` may improve performance.
+ *
+ * When set to `false`, the features will be maintained in an
+ * {@link ol.Collection}, which can be retrieved through
+ * {@link ol.source.Vector#getFeaturesCollection}.
+ *
```

Comment:
I would add that the spatial queries for extent/coordinate only work if set to true